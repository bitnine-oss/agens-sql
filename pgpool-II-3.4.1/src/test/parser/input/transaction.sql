BEGIN;
BEGIN WORK;
BEGIN TRANSACTION;
BEGIN ISOLATION LEVEL READ COMMITTED;
BEGIN ISOLATION LEVEL SERIALIZABLE;
BEGIN READ WRITE;
BEGIN READ ONLY;
START TRANSACTION;
COMMIT;
COMMIT WORK;
COMMIT TRANSACTION;
END;
END WORK;
END TRANSACTION;
ROLLBACK;
ROLLBACK WORK;
ROLLBACK TRANSACTION;
ABORT;
ABORT WORK;
ABORT TRANSACTION;
SET TRANSACTION ISOLATION LEVEL READ COMMITTED;
SET TRANSACTION ISOLATION LEVEL SERIALIZABLE;
SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL READ COMMITTED;
SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL SERIALIZABLE;
SET CONSTRAINTS ALL DEFERRED;
SET CONSTRAINTS ALL IMMEDIATE;
SET CONSTRAINTS x DEFERRED;
SET CONSTRAINTS x IMMEDIATE;
LOCK TABLE t1,t2;
LOCK t1,t2;
LOCK TABLE t1,t2 IN ACCESS SHARE MODE;
LOCK TABLE t1,t2 IN ROW SHARE MODE;
LOCK TABLE t1,t2 IN ROW EXCLUSIVE MODE;
LOCK TABLE t1,t2 IN SHARE UPDATE EXCLUSIVE MODE;
LOCK TABLE t1,t2 IN SHARE MODE;
LOCK TABLE t1,t2 IN SHARE ROW EXCLUSIVE MODE;
LOCK TABLE t1,t2 IN EXCLUSIVE MODE;
LOCK TABLE t1,t2 IN ACCESS EXCLUSIVE MODE;
