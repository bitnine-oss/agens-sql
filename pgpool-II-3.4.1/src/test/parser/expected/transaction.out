BEGIN 
BEGIN 
BEGIN 
BEGIN  ISOLATION LEVEL read committed
BEGIN  ISOLATION LEVEL serializable
BEGIN READ WRITE 
BEGIN READ ONLY 
START TRANSACTION 
COMMIT 
COMMIT 
COMMIT 
COMMIT 
COMMIT 
COMMIT 
ABORT 
ABORT 
ABORT 
ABORT 
ABORT 
ABORT 
SET TRANSACTION  ISOLATION LEVEL read committed
SET TRANSACTION  ISOLATION LEVEL serializable
SET SESSION CHARACTERISTICS AS TRANSACTION  ISOLATION LEVEL read committed
SET SESSION CHARACTERISTICS AS TRANSACTION  ISOLATION LEVEL serializable
SET CONSTRAINTS ALL DEFERRED
SET CONSTRAINTS ALL IMMEDIATE
SET CONSTRAINTS "x" DEFERRED
SET CONSTRAINTS "x" IMMEDIATE
LOCK TABLE "t1","t2" IN ACCESS EXCLUSIVE MODE
LOCK TABLE "t1","t2" IN ACCESS EXCLUSIVE MODE
LOCK TABLE "t1","t2" IN ACCESS SHARE MODE
LOCK TABLE "t1","t2" IN ROW SHARE MODE
LOCK TABLE "t1","t2" IN ROW EXCLUSIVE MODE
LOCK TABLE "t1","t2" IN SHARE UPDATE EXCLUSIVE MODE
LOCK TABLE "t1","t2" IN SHARE MODE
LOCK TABLE "t1","t2" IN SHARE ROW EXCLUSIVE MODE
LOCK TABLE "t1","t2" IN EXCLUSIVE MODE
LOCK TABLE "t1","t2" IN ACCESS EXCLUSIVE MODE
